chart/fertilesoil/templates/deployment.yaml

```diff
@@ -0,0 +1,139 @@
+    {{- if .Values.audit.enabled }}
+      initContainers:
+      # Optional: Pre-creates the `/app-audit/audit.log` named pipe.
+      - image: "{{ .Values.audit.auditImage.registry }}/{{ .Values.audit.auditImage.repository }}:{{ .Values.audit.auditImage.tag | default .Chart.AppVersion }}"
+        {{- with .Values.securityContext }}
+        securityContext:
+          {{- toYaml . | nindent 10 }}
+        {{- end }}
+        args:
+          - 'init'
+          - '-f'
+          - '/app-audit/audit.log'
+        name: init-audit-logs
+        imagePullPolicy: Always
+        volumeMounts:
+          - mountPath: /app-audit
+            name: audit-logs
+      {{- with .Values.audit.init.resources }}
+        resources:
+          {{- toYaml . | nindent 10 }}
+      {{- end }}
+    {{- end }}
+      containers:
+        - name: {{ template "common.names.fullname" . }}
+          {{- with .Values.securityContext }}
+          securityContext:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          image: "{{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          {{- with .Values.image.pullPolicy }}
+          imagePullPolicy: {{ . }}
+          {{- end }}
+          env:
+            - name: FERTILESOIL_CRDB_HOST
+              value: "{{ .Values.fertilsoil.env.FERTILESOIL_CRDB_HOST }}"
+            - name: FERTILESOIL_CRDB_USER
+              value: "{{ .Values.fertilsoil.env.FERTILESOIL_CRDB_USER }}"
+            - name: FERTILESOIL_CRDB_PARAMS
+              value: "{{ .Values.fertilsoil.env.FERTILESOIL_CRDB_PARAMS }}"
+            - name: FERTILESOIL_OIDC_AUDIENCE
+              value: "{{ .Values.fertilsoil.env.FERTILESOIL_OIDC_AUDIENCE }}"
+            - name: FERTILESOIL_OIDC_ISSUER
+              value: "{{ .Values.fertilsoil.env.FERTILESOIL_OIDC_ISSUER }}"
+            - name: FERTILESOIL_OIDC_JWKSURI
+              value: "{{ .Values.fertilsoil.env.FERTILESOIL_OIDC_JWKSURI }}"
+          args:
+            - "serve"
+            - "--nats-url"
+            - "{{ .Values.fertilsoil.nats.url }}"
+            - "--nats-nkey"
+            - "{{ .Values.fertilsoil.nats.nkey }}"
+            - "--auidit-log-path"
+            - "{{ .Values.fertilsoil.nats.audit_log_path }}"
+          ports:
+            - name: http
+              containerPort: {{ .Values.service.port }}
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /livez
+              port: http
+          readinessProbe:
+            httpGet:
+              path: /readyz
+              port: http
+          {{- with .Values.resources }}
+          resources:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+      {{- if .Values.audit.enabled }} # Begin audittail toggle
+      # This is mounted in the 1st container
+          volumeMounts:
+          - name: audit-logs
+            mountPath: /app-audit
+          - name: nkey-file
```

Comment:
this probably shouldn't be within the .Values.audit.enabled block since it's for nats.

Reply:
Nice catch, thanks!